controllers/projects.js

```diff
@@ -23,12 +31,63 @@ router.get('/new', function(req, res) {
   res.render('projects/new');
 });
 
+// POST from form action/projects/edit/<project.id>
+router.post("/edit/:id", function(req, res) {
+  // console.log(req.body);
+  db.project.update({
+    name: req.body.name,
+    githubLink: req.body.githubLink,
+    deployedLink: req.body.deployedLink,
+    description: req.body.description
+  }, {
+    where: {
+      id: req.params.id
+    }
+  })
+  .then(function(projId){
+    console.log(projId)
+    db.category.findOrCreate({
```

Comment:
... cont. here you do a find or create, so if the category exists great but if it doesn't then make it, there is no function to remove categories.